http_cache.rst

```diff
@@ -82,17 +82,17 @@ that wraps the default one (``AppKernel``). The caching Kernel *is* the reverse
 proxy.
 
 To enable caching, modify the code of your front controller. You can also make these
-changes to ``app_dev.php`` to add caching to the ``dev`` environment::
+changes to ``index.php`` to add caching to the ``dev`` environment::
 
-    // web/app.php
+    // public/index.php
     use Symfony\Component\HttpFoundation\Request;
 
     // ...
-    $kernel = new AppKernel('prod', false);
+    $kernel = new Kernel('prod', false);
     $kernel->loadClassCache();
```

Comment:
this line is obsolete?

Reply:
Reverted, all front controller and kernel changes should be done manually.